app/scripts/components/modellist.js

```diff
@@ -36,21 +45,23 @@ var exports = (function () {
         // every 10 seconds
         10000
       );
-
+      */
     },
     route: {
       data: function(transition) {
-        fetchModels()
-          .then(
-            (json) => {
-              // copy old data and set model
-              var data = this.$data;
-
-              data.models = json;
-              // transition to this new data;
-              transition.next(data);
-            }
-          );
+
+
+        // fetchModels()
+        //   .then(
+        //     (json) => {
+        //       // copy old data and set model
+        //       var data = this.$data;
+
+        //       data.models = json;
+        //       // transition to this new data;
+        transition.next();
+        //     }
+        //   );
```

Comment:
Again, I'd rather see code removed instead of commented, unless you have a comment to go with it (e.g. "An alternative method to do this would be...", or "This code is faster, but more prone to errors. See the docs @ http://...").